### PR TITLE
Cpurdy/#130

### PR DESCRIPTION
**Pull request by cpurdy**
#130 -> Marcus : this will need to be tested on macOS. It depends on the #129 changes. Wherever your current xec is in your path (assuming its not under your XDK_HOME itself), you should copy the new macos_launcher binary to that location. This change will now use the XDK_HOME location (if one is spec'd) intead of location where the launcher is in the path.

### CODE REVIEW

File: manualTests/build.gradle.kts [hunk 1]
```diff
@@ -187,9 +188,9 @@ xtcRun {
      * TODO: Add a nicer DSL syntax with a nested modules section.
      */
```
Comment (lagergren) [inline manualTests/build.gradle.kts:189]: You are doing something with the launcher in this branch, while I struggle to finish the build system. Can you please only modify the launcher code in this branch, and not do unrelated changes. I'm review ready in 30 minutes, and the manual tests do not look like that anymore.

Comment (lagergren) [reply manualTests/build.gradle.kts:189]: If you feel it is an important change that TestFizzBuzz changes names to EchoTest, please still do it in a separate small PR, so that it does not widen the impact area of conflict resolution.

**Review by ggleyzer — APPROVED** [review]
